cypress/integration/operations_spec.js

```diff
@@ -27,21 +27,7 @@ context('Fusion Studio', function () {
       cy.addCollection('admin@http://localhost:8080/db/test_col', 'col3');
     })
     it('create text document', function () {
-      cy.waitForLoading();
-      cy.getTreeNode('admin@http://localhost:8080/db/test_col').rightclick()
-      cy.getSubMenu('New document...').should('be.visible').trigger('mousemove')
-      cy.getMenuCommand('fusion.new-document').should('be.visible').click()
-      cy.get(dialogTitle).should('contain.text', 'New document');
-      cy.get(dialogBody).should('be.visible').find('input.theia-input[type=text]').should('contain.value', 'untitled').type('text_file.txt');
-      cy.get(dialogMainButton).should('be.visible').click();
-      cy.get(dialog).should('not.be.visible');
-      cy.getTreeNode('admin@http://localhost:8080/db/test_col/text_file.txt').should('be.visible');
-      cy.get('.p-Widget.p-TabBar li[title=' + CSS.escape('admin@http://localhost:8080/db/test_col/text_file.txt') + ']')
-      .should('be.visible').click()
-      .should('have.class', 'p-mod-current');
-      cy.get('[role=presentation].editor-scrollable').click().type('Sample text file content');
-      cy.get('#theia-top-panel .p-MenuBar-item').contains('File').click()
-      cy.get('.p-Menu-item[data-type=command][data-command=core\\.save]').click()
+      cy.addDocument('admin@http://localhost:8080/db/test_col', 'text_file.txt');
```

Comment:
I am not certain, but I vaguely recall that we removed most of these hard-coded strings like `admin@http://localhost:8080` and placed them into a config file as a set of variables? Perhaps it was related to Cypress? Can you remember where we did that?

Reply:
Don't worry I found them. They were not merged yet - https://github.com/evolvedbinary/fusion-studio-extension/pull/223